contributor-guide/modules/ROOT/pages/getting-involved.adoc

```diff
@@ -1,4 +1,60 @@
+
+Whichever list you add your issue to, the issue will be addressed in due course. Providing full details, and examples and screenshots where appropriate, should give you the quicker response.
+
+== Contribute to an Existing Library
+
+Rather than report an issue, or suggested improvement, and wait for another developer to address it, you can make the code changes yourself. To be successful you will need to be familiar with GitHub, and obviously a competent pass:[C++] developer. After finding the issue, or coming up with the improvement, you will need to go to the GitHub repository for that library, and create a fork and clone the library.
+
+With your fork of the library in your own GitHub account, you can make any changes and updates you wish. It is certainly recommended to thoroughly test the changes, and be aware of the test matrix for the library. You may have to test your changes on several operating systems, and with more than one compiler. It is also recommended you thoroughly comment your code changes, potentially every line of code you add should be commented as to its purpose.
+
+When you are satisfied you have met the bar for testing, you can commit the changes, push them to GitHub, and create a Pull Request. 
+
+After this, you can take a step back, and the owners or maintainers of the library will review your code. They might ask for some modifications or improvements. Incorporate the feedback provided by the reviewers and push your modifications.
+
+If all goes well, your changes will be merged by the library admins, and you will have made a welcome contribution.
+
+== Engage in the Review Process
+
+There are several roles in the review process, from managing the steps of the review process, to testing and providing feedback. Read the xref:formal-reviews:ROOT:index.adoc[] for details.
+
+== Maintain an Existing Library
+
+Maintaining an existing library is similar in steps to <<3. Contribute to an Existing Library>>, in that you will have to create a fork and clone the library. However, more is expected. You will have to spend time educating yourself on the full functionality of the library, and be prepared to make changes to the code anywhere changes are needed. 
+
+For a larger library this education can turn out to be a significant, and perhaps onerous task. It can be quite difficult to reverse engineer in your mind the processes and purpose of code written by another developer, with all the expected personal style and idiosyncrasies that make their way into source code. However, that being said, it can be a valuable contribution to take an existing but deprecated library you are interested in and give it a new lease of life.
```

Comment:
Note that unmaintained != deprecated. There are useful libraries that went unmaintained because the author didn't have time for them.

Reply:
I would make extra distinctions:

discontinued < deprecated < unmaintained < poorly maintained < maintenance mode / no new features < not open to contributions < open to contributions

Nothing is really discontinued, deprecated, or unmaintained in Boost. Everything is tested in CI and the super-project is always coherent. We could have a policy for deprecating libraries but we don't really have one yet. I'm not sure a policy for discontinuing libraries makes much sense in practice because we would be breaking other people's code for very little benefit.

A few libraries are poorly maintained for many reasons. They might need significant changes for which we don't have the human resources.

Most libraries people say are unmaintained are actually in basic maintenance mode. We make sure it works on new compilers and it's coherent with the super-project, but there's no point in adding features to things that went to the standard, such as boost::optional, or providing replacements, such as coroutine and coroutine2.

The most important distinction to contributors is probably not open to contributions != open to contributions, because we want to provide users with a way to figure it out so that they don't write PR or issues that go nowhere.

Reply:
OK, added several sentences to Issues, and Maintaining Libraries, to verify with the library admins that they are open to updates.